packages/@sfx/sayt-driver-plugin/src/sayt-driver-plugin.ts

```diff
@@ -181,59 +199,24 @@ export default class SaytDriverPlugin implements Plugin {
 
   /**
    * Extracts query and products from the given response.
+   * Calls `this.transformProduct` on each product found in the response.
+   * Filters out any products that map to a falsy value.
```

Comment:
We can save this for a documentation overhaul, but these three sentences together flow awkwardly. Consider joining the last two sentences and using a pronoun. Something like `It calls [[transformProduct]] on each product found in the response and filters out any that map to a falsy value.`.